specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/inference.json

```
@@ -0,0 +1,1443 @@
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/InferenceEndpointCheckNameAvailabilityParameter"
+            },
+            "description": "The name of the endpoint within the specified resource group."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Operation to check the endpoint name availability was successful.",
+            "schema": {
+              "$ref": "#/definitions/CheckNameAvailabilityResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/inferenceEndpoints/{inferenceEndpointName}/deployments": {
+      "get": {
+        "tags": [
+          "MachineLearningInferenceEndpointDeployments"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "operationId": "MachineLearningInferenceEndpointDeployments_ListByInferenceEndpoint",
+        "description": "Gets Inference Deployments.",
+        "x-ms-examples": {
+          "Get Deployments": {
+            "$ref": "./examples/getDeployments.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/PaginationParameter"
+          },
+          {
+            "$ref": "#/parameters/InferenceEndpointNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The response includes a paginated array of Machine Learning Inference Deployments.",
+            "schema": {
+              "$ref": "#/definitions/PaginatedInferenceEndpointDeploymentsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/inferenceEndpoints/{inferenceEndpointName}/deployments/{deploymentName}": {
+      "get": {
+        "tags": [
+          "MachineLearningInferenceEndpointDeployments"
+        ],
+        "operationId": "MachineLearningInferenceEndpointDeployment_Get",
```

Align with the List operationId
```suggestion
        "operationId": "MachineLearningInferenceEndpointDeployments_Get",
```